examples/lesbrary_catke_calibration.jl

```diff
@@ -0,0 +1,195 @@
+
+priors = (Cᴰ   = lognormal_with_mean_std(2.0, 0.5),
+          Cᵂu★ = lognormal_with_mean_std(2.0, 0.5),
+          Cᴸᵇ  = lognormal_with_mean_std(0.2, 0.1),
+          Cᴷu⁻ = ConstrainedNormal(0.2, 0.1, 0.0, 2.0),
+          Cᴷc⁻ = ConstrainedNormal(1.0, 0.1, 0.0, 2.0),
+          Cᴷe⁻ = ConstrainedNormal(1.5, 0.3, 0.0, 2.0))
+
+free_parameters = FreeParameters(priors)
+
+# TODO: explain the meaning of each parameter
+# The prior information comes from experience, prior calibration runs,
+# and educated guesses.
+
+calibration = InverseProblem(observations, simulation, free_parameters)
+
+# Next, we calibrate, using a relatively large noise to reflect our
+# uncertainty about how close the observations and model can really get,
+
+eki = EnsembleKalmanInversion(calibration;
+                              noise_covariance = 1e-2,
+                              resampler = NaNResampler(abort_fraction=0.5))
+
+iterate!(eki; iterations = 10)
+
+# # Results
+#
+# To analyze the reuslts, we build a new simulation with just one ensemble member
+# to evaluate pasome utilities for analyzing the results:
+
+Nt = length(observations.times)
+Niter = length(eki.iteration_summaries) - 1
+modeled_time_serieses = calibration.time_series_collector.field_time_serieses 
+observed = map(name -> observations.field_time_serieses[name][Nt], field_names)
+modeled = map(name -> modeled_time_serieses[name][Nt], field_names)
+
+function compare_model_observations(model_label="modeled")
+    fig, axs = make_figure_axes()
+    plot_fields!(axs, observed..., "observed at t = " * prettytime(times[end]), :black)
+    plot_fields!(axs, modeled..., model_label, :blue)
+    [axislegend(ax, position=:rb, merge=true, fontsize=10) for ax in axs]
+    return fig
+end
+
+# Now we execute forward runs for the initial ensemble mean,
+
+initial_parameters = eki.iteration_summaries[0].ensemble_mean
+forward_run!(calibration, initial_parameters)
+fig = compare_model_observations("modeled after 0 iterations")
+
+## display(fig)
+save("model_observation_comparison_iteration_0.svg", fig); nothing # hide
+
+# ![](model_observation_comparison_iteration_0.svg)
+
+# and the final ensemble mean, representing our "best" parameter set,
+
+best_parameters = eki.iteration_summaries[end].ensemble_mean
+forward_run!(calibration, best_parameters)
+fig = compare_model_observations("modeled after $Niter iterations")
+
+## display(fig)
+save("model_observation_comparison_final_iteration.svg", fig); nothing # hide
+
+# ![](model_observation_comparison_final_iteration.svg)
+
+# ## Parameter evolution
+#
+# To understand how results changed over the EKI iterations,
+# we look at the evoluation of the ensemble means,
+
+ensemble_means = NamedTuple(n => map(summary -> summary.ensemble_mean[n], eki.iteration_summaries)
+                            for n in calibration.free_parameters.names)
+
+fig = Figure()
+ax = Axis(fig[1, 1], xlabel = "Ensemble Kalman iteration", ylabel = "Parameter value")
```

Comment:
Are all these parameters diffusivities? Then perhaps:

```suggestion
ax = Axis(fig[1, 1], xlabel = "Ensemble Kalman iteration", ylabel = "Parameter value [m² s⁻¹]")
```

Reply:
No, they are non-dimensional.